src/lms-ds-loader/edfi_lms_ds_loader/scripts/mssql/create_section_tables.sql

```diff
@@ -0,0 +1,61 @@
+-- SPDX-License-Identifier: Apache-2.0
+-- Licensed to the Ed-Fi Alliance under one or more agreements.
+-- The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+-- See the LICENSE and NOTICES files in the project root for more information.
+
+CREATE TABLE lms.LMSSection (
+    LMSSectionIdentifier INT NOT NULL IDENTITY,
+    SourceSystemIdentifier NVARCHAR(255) NOT NULL,
+    SourceSystem NVARCHAR(255) NOT NULL,
+    SISSectionIdentifier NVARCHAR(255) NULL,
+    Title NVARCHAR(255) NOT NULL,
+    SectionDescription NVARCHAR(1024) NULL,
+    Term NVARCHAR(60) NULL,
+    LMSSectionStatus NVARCHAR(60) NULL,
+    SourceCreateDate DATETIME2(7) NULL,
+    SourceLastModifiedDate DATETIME2(7) NULL,
+    Discriminator NVARCHAR(128) NULL,
+    CreateDate DATETIME2 NOT NULL,
+    LastModifiedDate DATETIME2 NOT NULL,
+    DeletedAt DATETIME2(7) NULL,
+    CONSTRAINT LMSSection_PK PRIMARY KEY CLUSTERED (
+        LMSSectionIdentifier ASC
+    ) WITH (PAD_INDEX = OFF, STATISTICS_NORECOMPUTE = OFF, IGNORE_DUP_KEY = OFF, ALLOW_ROW_LOCKS = ON, ALLOW_PAGE_LOCKS = ON) ON [PRIMARY]
+) ON [PRIMARY];
+
+ALTER TABLE lms.LMSSection ADD CONSTRAINT LMSSection_DF_CreateDate DEFAULT (getdate()) FOR CreateDate;
+ALTER TABLE lms.LMSSection ADD CONSTRAINT LMSSection_DF_LastModifiedDate DEFAULT (getdate()) FOR LastModifiedDate;
+ALTER TABLE lms.LMSSection ADD CONSTRAINT LMSSection_UK_SourceSystem UNIQUE (SourceSystemIdentifier, SourceSystem);
+
+EXEC sys.sp_addextendedproperty @name=N'MS_Description', @value=N'An organized grouping of course content and users over a period of time for the purpose of providing instruction.', @level0type=N'SCHEMA', @level0name=N'lms', @level1type=N'TABLE', @level1name=N'LMSSection';
```

Comment:
I don't remember the syntax for these. But we should be able to copy these straight from the MetaEd output.